### DISCUSSION TITLE
How to locate the comment text in the document?

### DISCUSSION THREAD

**Discussion by NangeGao** [outline/outline#6131] [General] [answered]
I find sometimes when in a long article, some comments in the right side cannot link to the origin overline text in the document. I check the request about how to create one comment, when call `/api/comments.create`, the request payload as below. There seems no location info in the request body. I am very confused about how to render the comment's location in the document. Please help me.

```
{
   "documentId":"7f1f7bc2-63c3-4cd7-b40e-0d17161731b3",
   "data":{
      "type":"doc",
      "content":[
         {
            "type":"paragraph",
            "content":[
               {
                  "type":"text",
                  "text":"Hello world !"
               }
            ]
         }
      ]
   },
   "id":"bd438b64-a771-4a72-b4eb-59575984f2cc"
}
```

**Comment by tommoor** [comment]
It is possible to create comments that are not linked to any text from the bottom of the comment sidebar, are you sure this isn't what's happening here?

**Reply by NangeGao** [reply]
I know this. In fact I need the marking comment mode. But if the original marked text is disapeared, my workmates may cannot understand me clearly. I am trying to find the reason and fix it.

**Reply by tommoor** [reply]
Did the actual text that was highlighted disappear, or just the connection to the comment?

**Reply by NangeGao** [reply]
The text exists, the highlighted marker disappears, so no connection at all.

**Reply by NangeGao** [reply]
https://github.com/outline/outline/discussions/5985  I met the same as this.

**Reply by tommoor** [reply, marked as answer]
This was fixed in https://github.com/outline/outline/issues/6011 – will be in the next patch release soon.
